src/App.tsx

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import HomePage from "./pages/HomePage/HomePage";
 import SearchPage from "./pages/SearchPage/SearchPage";
+import HawkersPage from "./pages/HawkersPage/HawkersPage";
+import IndividualHawker from "./pages/IndividualHawker/IndividualHawker";
 import './App.css';
 import 'semantic-ui-css/semantic.min.css'
 
 function App() {
   return (
     <Router>
       <Switch>
+        <Route path="/individualhawker">
```

Comment:
Should probably name it different. Perhaps use the name of the hawker centre?

Reply:
Should have wrapped around the hawkers path so that we get `/hawkers/name-of-hawker-centre`